README.md: Add Arabic translation to the Translations section

Jamil-Bailony opened the pull request:
Hello Donovan,
We would like to add link to the translated version of the book in the book's repo. Unfortunately, we cannot add the translated pages to the repo because the RTL direction is not supported here in GitHub.
 
Thank you very mush.

jBailony committed:
README.md: Add Arabic translation to the Translations section

donovanh commented:
Thanks for doing this! I'd much prefer you include the PDF file in the pull request rather than link to a private website. 

Would you mind putting the PDF in a `downloads/translations/arabic` folder?

If you like I'd be happy for you to have credit for doing the translation. You could add something like `Translated by [@HsoubAcademy](http://academy.hsoub.com)` after the PDF download link in the README if that makes sense?

Commit:
downloads: Upload  Arabic PDF version and add a link in README.md

Jamil-Bailony commented:
Hi @donovanh,

I've added the PDF version to the `downloads/translations/arabic` folder and updated the related link in the `README.md` file.

Thank you very much for your suggestion. We appreciate your help.
